index2.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Morse Code Translator</title>
+  <link rel="stylesheet" type="text/css" href="./styles2.css">
+  <script src="./script2.js" defer></script>
+</head>
+<body>
+  <h1>Morse Code Translator</h1>
+
+  <div class="translation-container">
+    <textarea id="input-text" placeholder="Enter text or Morse code"></textarea>
+    <div class="buttons">
+      <button id="translate-to-morse">Translate to Morse</button>
+      <button id="translate-to-text">Translate to Text</button>
```

Comment:
maybe it would be worth adding a "Clear" button to reset the inputs, think of having just a "Translate" btn, check using regex if it's only . _ /, then its more code